validation/custom_constraint.rst

```diff
@@ -59,11 +59,20 @@ The validator class is also simple, and only has one required method ``validate(
 
     use Symfony\Component\Validator\Constraint;
     use Symfony\Component\Validator\ConstraintValidator;
+    use Symfony\Component\Validator\Exception\UnexpectedTypeException;
 
     class ContainsAlphanumericValidator extends ConstraintValidator
     {
         public function validate($value, Constraint $constraint)
         {
+            if (null === $value || '' === $value) {
+                return;
```

Comment:
What about adding a commented code line for putting the reason why we must do so?